backend/app.js

```diff
@@ -49,7 +50,7 @@ app.use(morgan("dev"));
 // app.use(helmet());
 
 // Cross-Origin-Resource-Sharing
-// app.use(cors());
+app.use(cors());
```

Comment:
We shouldn't allow CORS for all APIs, so please remove changes from backend/app.js.